### DISCUSSION TITLE
最新的桌面客户端2.9.9无法使用

### DISCUSSION THREAD

**Discussion by XlangNan** [1715173329/UnblockNeteaseMusic#658] [Q&A] [answered]
最新的桌面客户端2.9.9无法使用了，测试代理是通的但是加载歌曲就显示网络失败。

**Comment by 1715173329** [comment, marked as answer]
证书信任了吗？endpoint 设定了吗？
测试代理无效才是正常工作的表现。

**Comment by XlangNan** [comment]
> 证书信任了吗？endpoint 设定了吗？ 测试代理无效才是正常工作的表现。

可是 我一直这样做啊 就用的 http代理，安卓手机也是用clash 写的配置文件  网易云音乐极速版 就可以正常使用

**Comment by 1715173329** [comment]
那就去用旧版的客户端。

**Comment by XlangNan** [comment]
> 那就去用旧版的客户端。



> 证书信任了吗？endpoint 设定了吗？

证书信任 endpoint 设定 有详细的教程吗  不甚感激

**Comment by YukimiNan** [comment]
[安裝 UNM 的 HTTPS 憑證 #426](https://github.com/UnblockNeteaseMusic/server/discussions/426)

**Comment by w6lc** [comment]
我是2.98的，也是测试代理通过但是连接失败，把service mode卸载重装，重启，莫名奇妙就好了😥

**Reply by XlangNan** [reply]
service mode 是什么 在哪里卸载的

**Reply by w6lc** [reply]
可能我们不是一种情况，我是用的轻量模式，需要安装service mode，主界面就有。
默认是不会安装的
